seismic/inversion/wavefield_decomp/example_wf_continuation_tao.py

```diff
@@ -79,11 +80,12 @@ def example_4():
     rho = [rho_c]
     fixed_args = (flux_comp, mantle_props, Vp, rho, FLUX_WINDOW)
     H_min, H_max = (25.0, 45.0)
-    bounds = optimize.Bounds([H_min, Vp_c/k_max], [H_max, Vp_c/k_min])
+    bounds = optimize.Bounds(np.array([H_min, Vp_c/k_max]),
+                             np.array([H_max, Vp_c/k_min]))
 
     # - Basin hopping
     # logging.info('Trying basinhopping...')
-    # soln_bh = optimize.basinhopping(objective_fn, model_initial_poor, T=0.3,
+    # soln_bh = optimize.basinhopping(objective_fn, model_initial_poor, T=0.5, stepsize=1.0,
```

Comment:
It would be good to have hardwired parameters somewhere in separate file such as header file ? Addition of value explanation would help to read the code.

Reply:
I will try to centralize common parameters where possible. In this case these parameters are problem specific, tuned to get good coverage of parametric space. I do not consider these to be 'magic numbers' because they are assigned to a particular function argument which is already heavily documented.